tests/test_datafiles.py

```diff
@@ -0,0 +1,30 @@
+"""Test installation of PINT data files"""
+import os
+import unittest
+import pint
+
+
+class TestDatafiles(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+
+        # get a list of par and tim files to make sure we can find
+        files = """B1855+09_NANOGrav_9yv1.gls.par
+        B1855+09_NANOGrav_9yv1.tim
+        B1855+09_NANOGrav_dfg+12.tim
+        B1855+09_NANOGrav_dfg+12_TAI.par
+        J0613-sim.par
+        J0613-sim.tim
+        J1614-2230_NANOGrav_12yv3.wb.gls.par
+        J1614-2230_NANOGrav_12yv3.wb.tim
+        NGC6440E.par
+        NGC6440E.tim
+        PSRJ0030+0451_psrcat.par
+        waves.par
+        waves_withpn.tim"""
+
+        cls.files = [f.strip() for f in files.split("\n")]
+
+    def test_datafiles(self):
```

Comment:
Maybe have a test that uses `try: ... finally:` to switch to a temporary directory and see if it can still access the files? Like, is our retrieval mechanism going to work okay from unexpected directories? If one were especially distrustful one could try retrieving the filename, changing the directory, and then opening the file. But I do think it's important we check this mechanism doesn't rely on any particular location being the current directory.

Reply:
I don't understand why this needs a `try...`. But I have added tests to change to temp directories both before and after determining the file names.

Reply:
The try/finally is to ensure that the current working directory is restored whether the test succeeds or fails, or if some other event (control-C for example) occurs in the middle of it. Ideally there would be a context manager to do this but I haven't found one in the Python standard library.